workflows/post_process_run/scream_post/format_output.py

```diff
@@ -0,0 +1,186 @@
+from typing import Hashable, List, Mapping
+import xarray as xr
+import numpy as np
+import os
+
+output_variables = [
+    "T_mid",
+    "qv",
+    "omega",
+    "horiz_winds",  # this will be changed to u and v in the future
+    "surf_sens_flux",
+    "surf_evap",
+    "SW_clrsky_flux_up@bot",
+    "SW_clrsky_flux_dn@bot",
+    "LW_clrsky_flux_dn@bot",
+    "LW_clrsky_flux_up@bot",
+    "SW_clrsky_flux_up@tom",
+    "LW_clrsky_flux_up@tom",
+    "VerticalLayerInterface",
+    "p_mid",
+    "ps",
+    "VerticalLayerInterface",
+    # "pseudo_density",  # this is pressure thickness
+    "area",
+    "lat",
+    "lon",
+    "hyai",
+    "hybi",
+]
+
+NP = 4
+NPG = 2
```

Comment:
I guess this has some standard meaning for SCREAM, but I have no idea what it represents. Either give a more useful variable name (preferred) or add a short comment.

Reply:
No longer need these